src/app/app.module.ts

```diff
@@ -1,22 +1,96 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
+import { MSAL_INSTANCE, MSAL_GUARD_CONFIG, MSAL_INTERCEPTOR_CONFIG } from '@azure/msal-angular';
+
+import {
+  MsalModule,
+  MsalService,
+  MsalInterceptor,
+  MsalInterceptorConfiguration,
+  MsalGuard,
+  MsalGuardConfiguration,
+  MsalBroadcastService, 
+  MsalRedirectComponent
+} from "@azure/msal-angular";
+import { IPublicClientApplication, PublicClientApplication, InteractionType, BrowserCacheLocation } from "@azure/msal-browser";
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { HeaderComponent } from './layout/header/header.component';
 import { FooterComponent } from './layout/footer/footer.component';
+import { environment } from '../environments/environment';
+import { UserService } from './user-service';
+
+export function MSALInstanceFactory(): IPublicClientApplication {
+  return new PublicClientApplication({
+    auth: {
+        clientId: environment.VTM_CLIENT_ID,
+        authority: environment.VTM_AUTHORITY_ID,
+        redirectUri: '/',
+
+    },
+    cache: {
+      cacheLocation: BrowserCacheLocation.LocalStorage,
+      storeAuthStateInCookie: true,
+    },
+  });
+}
+
+export function MSALInterceptorConfigFactory(): MsalInterceptorConfiguration {
+  const protectedResourceMap = new Map<string, Array<string>>();
+  protectedResourceMap.set("https://graph.microsoft.com/v1.0/me", ["user.read"]);
```

Comment:
Is eslint not pickup on the double quotes?

Reply:
I'm currently working on getting eslint working properly :)

Reply:
I'm just going to leave these linter comments, as waiting on the linter to work.